3D Chess/Assets/Scripts/PieceEntity.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Realms;
+using UnityEngine;
+
+public class PieceEntity : RealmObject
+{
+    public int Type { get; set; }
+
+    public float PositionX { get; set; }
+    public float PositionY { get; set; }
+    public float PositionZ { get; set; }
+
+    public PieceEntity()
+    {
+
+    }
+
+    public PieceEntity(PieceType type, Vector3 position)
+    {
+        Type = (int)type;
+        PositionX = position.x;
+        PositionY = position.y;
+        PositionZ = position.z;
```

Comment:
This can call `SetPosition` or eventually `Position = position`.

Reply:
See 6d79643.

Reply:
This can now use the convenience properties:

```csharp
PieceType = type;
Position = position;
```

Reply:
True, that part was missing. Added with 5ab5fb1.